### DISCUSSION TITLE
Adding image preview to article list using image in page bundle?

### DISCUSSION THREAD

**Discussion by EdwardDeaver** [jpanther/congo#315] [Q&A]
Hello, 

I am trying to add the ability to have a image preview for my article in the Article-Link.html. 

I can't figure out how to add an image that is in a page bundle. Like what is the URL of it?

So if I have a bundle:

-posts
-- MyCoolPost
   ---- cats.jpg

How do I query that cats photo?

I made a front matter parameter that is a bool for if there is a thumbnail and one for the image path. 
  {{ if   .Params.thumbnail | default false}}
  {{$base := .File.Dir}}
{{$foo := .Params.thumbnailLink }}
{{$url := print .Site.Params.baseURL "/" $base $foo}}

{{ end }}

but I can't get that cats photo.

**Comment by jpanther** [comment]
This is now available in Congo v2.4.0. Just add an image to the page bundle with "thumb" in the name and it will be included in the article link. The [getting started docs](https://jpanther.github.io/congo/docs/getting-started/#feature-cover-and-thumbnail-images) have more details on how this new feature works.

**Reply by EdwardDeaver** [reply]
Hi @jpanther , 

That's awesome, thank you! I do have a question about it (I'm not that familiar with tailwind). Is it possible to increase the image size?

Best,
Edward

**Reply by jpanther** [reply]
It's possible to change this by providing your own `list.html` or `single.html` templates which override the default behaviour, but it's not possible to simply change a configuration value to make the images larger.
